54 refactoring

woikal opened the pull request:
Untested!
Seperation of Beans, Services into packages, seperate (Web)Controllers
Missing: visibility of field may be changed.

SecMeyo committed:
12-refactor-master merged

SecMeyo committed:
12-refactor-master merged

Yolgie committed:
12-refactor-apply stash after mega merge from master

SecMeyo committed:
12-refactor-imports fixed

SecMeyo committed:
12-refactor-imports fixed

SecMeyo committed:
12-refactor-jkulan package removed

SecMeyo committed:
12-refactor-jkulan package removed

SecMeyo committed:
12-fix-added null cases

SecMeyo committed:
12-fix-added null cases

Yolgie committed:
50-feature-created date, modified date, generalized actionlog

woikal committed:
Integration of scss transcription done
- new gradle task "sassCompile"
- automated transcription on build
- manuel transcription on bootRun necessary

Yolgie committed:
12-feature-timeService, tournament controller

fuero committed:
Merge pull request #52 from Yolgie/scss_support

Integration of scss transcription done

fuero committed:
Merge pull request #51 from Yolgie/50-audited-db-entity

50-feature-created date, modified date, generalized actionlog

SecMeyo committed:
12-feature-cross reference users-teams

SecMeyo committed:
12-fix-@PermitAll removed

Yolgie committed:
12-feature-tournament controller create and list

Yolgie committed:
Merge remote-tracking branch 'origin/12-Tournament-Manager' into 12-Tournament-Manager

Yolgie committed:
Merge pull request #54 from Yolgie/12-Tournament-Manager

12 tournament manager

Commit:
Move DAOs to dao package

Commit:
Change SteamId type to long

Commit:
adds support to query Steam data, adds unit tests

 - adds code to query Steam API for game and user data
 - adds support to store Steam game data to the database
 - adds unit testing support (dependencies and integration in Spring
   boot)
 - adds skeleton code to populate the DB with data from Steam

Commit:
fix imports

woikal committed:
54-package base

woikal committed:
54-method pars in UserService, make non-static

woikal committed:
54-TeamService, TeamController

Commit:
Steam Sync working, basic junit tests added

woikal committed:
54-TournamentService, TournamentController
Inheritance from EventService/Event

UNTESTED

Commit:
54-refactoring: util/Validation

Yolgie commented:
:+1:

Yolgie commented:
Hier werden allerdings in der GUI viel mehr Änderungen angezeigt als notwendig...

woikal commented:
wie meinen?

TTH-Someone-stole-my-name committed:
Renamed converter.java to start with capital C.

TTH-Someone-stole-my-name committed:
Extended Validator to use standard Java Mail Validation.

Commit:
Missed junit dependency in non-test code

SecMeyo commented:
rename converter.java > Converter.java
im Output seh ich keinen Stylesheet mehr, soll das so?

woikal committed:
WIP debugging jUnit failure on TestSteamSync (persisting auto thingy..)

TTH-Someone-stole-my-name committed:
Removed UserState from Hibernate Entities. Keeping this in a DB is a bad solution in my mind. Better work with cached Methods here.

Added GameRepo to reliably save Game's. Don't ask me why this does not work in other scenarios.

Updated some tests to reflect the missing UserState Object.

TTH-Someone-stole-my-name committed:
Merge remote-tracking branch 'remotes/origin/jUnit-debug' into 54-refactoring

Merged the jUnit-Debug Branch (which originated from @fuero's currently-played branch).
I did make some changes to it however, so I thought it better to leave the original currently-played branch untouched.

TTH-Someone-stole-my-name committed:
54-refactoring-cleanup: Removed in my opinion unncecessary complexity in updating Steam information. Also extended Services to needs risen through SteamIntegration.

TTH-Someone-stole-my-name committed:
54-refactoring-cleanup: Some other minor cleanups.

TTH-Someone-stole-my-name committed:
54-refactoring-Fix: Fixed access to name in MyUserDetailsService to enable Google Login.

TTH-Someone-stole-my-name commented:
OK, ich hab mich jetzt mal hingesetzt und diesen Branch mit dem jUnitDebug Branch kombiniert --> Jetzt haben wir da drinnen ein paar Beispiele für funktionierende UnitTests. Allerdings hab ich dabei auch einige neue Changes eingearbeitet --> Ihr werden den Branch also noch ein zweites Mal reviewen müssen. Ich werd heute noch aufm Master rebasen, damit wir den auch automatisch mergen können.

TTH-Someone-stole-my-name committed:
Merge remote-tracking branch 'remotes/origin/master' into 54-refactoring

Merge done manually to test before signing merge request

TTH-Someone-stole-my-name committed:
54-refactoring: Post Merge fixes.

woikal commented:
:+1:
Hab das refoctory-refit von TTH jetzt nochmal eingehend durchgesehn und würd es so übernehmen.
